packages/flame-flags/README.md

```diff
@@ -0,0 +1,308 @@
+# Flame Flags
+
+Flags used across LightSpeed products
+
+This package only provides the raw SVG files as well as a spritesheet.
+
+## How to use individual flags
+
+Each flags are available within the `src` directory. Flag names must use their ISO country code.
+It is also important to update the list of `flag-iso.list.json` with the new flag ISO and it's corresponding
+country.
+
+If you are within a context where you can import SVGs via import statements (you may need to include https://webpack.js.org/loaders/svg-inline-loader/), you may do the following:
+
+```js
+import CanadaFlagSVG from '@lightspeed/flame-flags/src/CA.svg';
```

Comment:
Unless you want to have a bunch of icons at the root of the package, nothing prevents us from moving everything back up.

This is mostly done for consistency purposes (all other packages uses the `src` directory). We can do an exception here, but we can also argue further on moving icons into 1 single package (instead of having icons and flags) with "buckets". So, we'd put icons in the icons folder and such, so we'd end up with...

```
import CanadaFlagSVG from '@lightspeed/flame-svgs/flags/CA.svg';
```

Which could also work and may make more sense